GitUI/ScriptsEngine/ScriptInfo.cs

```diff
@@ -34,12 +34,38 @@ public ScriptInfo()
         /// </summary>
         public string? Icon { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Icon or associated file path
```

Comment:
I'm finding this confusing. This property isn't "setting Icon" or anything other property. 
If this property contains the path to the file that contains the icon then
```suggestion
        /// Gets or sets the path to the file containing the icon.
```